packages/kbn-search-connectors/components/cron_editor/enterprise_search_cron_editor.tsx

```diff
@@ -7,9 +7,7 @@
 
 import React, { useState } from 'react';
 
-import { Frequency } from '@kbn/es-ui-shared-plugin/public/components/cron_editor/types';
```

Comment:
Not sure why `Frequency` was imported from `@kbn/es-ui-shared-plugin/public/components/cron_editor/types`, when we have redefined Frequency . Added this change in a `packages/kbn-search-connectors/components/cron_editor/cron_editor.test.tsx` too.